authors/apps/articles/views.py

```diff
@@ -43,6 +43,24 @@ class ArticleView(RetrieveUpdateDestroyAPIView):
     """
     permission_classes = (IsAuthenticatedOrReadOnly,)
 
+    def get(self, request, slug):
+        """
+        Retrieve a specific article
+        :param slug
+        any user should view details of an article
+        """
+        # Get the article user searched
+        # When article is not found, Give the user a description
+        # of what happened. If the article is found return it back to user
+        try:
+            article = Article.objects.get(slug=slug)
+        except Article.DoesNotExist:
+            raise exceptions.NotFound({
+                "message": "Article was not found"})
+        return Response(
```

Comment:
I believe it would be more easier to use serializer and JSON render to handle the return values. This will make it easier to get the authors' profile data which should also be included in the return response, which currently is not.